web-registry/codegen.yml

```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: 'http://localhost:5001/graphql'
+schema: 'http://localhost:5000/graphql'
```

Comment:
@blushi I noticed this was changed in one of your [recent commits](https://github.com/regen-network/regen-web/commit/2e4d3e99993714d473a4f657e2d9db79d456a172) - I thought I recalled you giving a reason, but can't find in git.

I changed it back to 5000 here because that's the default port for our GraphQL server, but want to make sure you see this change in case I need to keep at 5001

Reply:
thanks for that!
there was no particular reason, apart from the fact that I was already using 5000 locally for something else, so was using 5001 temporarily for the registry-server and forget to revert the changes on regen-web